### DISCUSSION TITLE
Compatibility Issue: @capawesome/capacitor-app-update with @capacitor/core 6.0.0-rc.0

### DISCUSSION THREAD

**Discussion by blaiseAI** [capawesome-team/capacitor-plugins#140] [Q&A] [answered]
Hello, community,

I've encountered a compatibility error when trying to use @capawesome/capacitor-app-update alongside @capacitor/core 6.0.0-rc.0. The exact error message is as follows:

`[error] Your version of @capawesome/capacitor-app-update is not compatible with @capacitor/core 6.0.0-rc.0.`

This issue arises when attempting to update or integrate the @capawesome/capacitor-app-update package in a project that's currently using the release candidate version 6.0.0-rc.0 of @capacitor/core.

I've checked the documentation for both packages but haven't found any guidance on compatibility concerns or specific version requirements that address this error. I'm seeking advice on how to resolve this compatibility issue. Specifically, I'm looking for:

Any known compatible versions of @capawesome/capacitor-app-update that work with @capacitor/core 6.0.0-rc.0, or vice versa.
Recommendations on how to configure or update my project to mitigate this compatibility error.
Insights into whether this is a known issue with a pending fix or if there are any workarounds available.
I've attempted to revert to previous versions of both packages without success and am cautious about downgrading @capacitor/core significantly due to other dependency requirements in my project.

Thank you in advance for any assistance or guidance you can provide!

**Comment by robingenz** [comment, marked as answer]
As you can see in #103, Capacitor 6 support is not yet available.
